There is an error in this problem: 
	- File path: ../../scripts/webwork/source/College_Physics_Urone/29.Introduction_to_Quantum_Physics/29-08.The_Particle_Wave_Duality_Reviewed/NU_U17_29_08_004.md
	- Error: 'part3'


Traceback (most recent call last):
  File "process.py", line 63, in main
    pbs.process_question_pl(source_filepath)
  File "/Users/parsaimac/.pyenv/versions/3.8.5/lib/python3.8/site-packages/problem_bank_scripts/problem_bank_scripts.py", line 615, in process_question_pl
    q_type = parsed_q['header'][part]['type']
KeyError: 'part3'
